commit 4f87d08987ab6332154cc37f81019de15a4af894
Author: Ritesh Raj Sarraf <rrs@researchut.com>
Date:   Fri Mar 16 01:23:40 2012 +0530

    tighter checks
    
    Please see http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=654262 for
    details. Thanks to Aurelien Jarno
    
    Signed-off-by: Ritesh Raj Sarraf <rrs@researchut.com>
    Signed-off-by: Nicholas Bellinger <nab@risingtidesystems.com>

diff --git a/scripts/rc.target b/scripts/rc.target
index 22c0350..7ad75a3 100644
--- a/scripts/rc.target
+++ b/scripts/rc.target
@@ -51,9 +51,9 @@ LIO_CONFIGFS_SCRIPT="/etc/target/lio_start.sh"
 # Enable: Set DEBUG=1    Disable: Set DEBUG=0
 
 DEBUG=0
-LOGFILE=/tmp/tgtctl.dbug
+LOGFILE=/var/log/tgtctl.dbug
 
-if [ $DEBUG ]; then
+if [ $DEBUG != 0 ]; then
 	echo "$0 $*" >> $LOGFILE
 fi
 #########################################################################
